environment.yml

```diff
@@ -0,0 +1,115 @@
+libffi                    3.3                  he6710b0_2  
+libgcc-ng                 9.1.0                hdf63c60_0  
+libgfortran-ng            7.3.0                hdf63c60_0  
+libpng                    1.6.37               hbc83047_0  
+libstdcxx-ng              9.1.0                hdf63c60_0  
+libtiff                   4.1.0                h2733197_1  
+libuuid                   1.0.3                h1bed415_2  
+libxcb                    1.14                 h7b6447c_0  
+libxml2                   2.9.10               he19cac6_1  
+lz4-c                     1.9.2                he6710b0_0  
+markupsafe                1.1.1            py38h7b6447c_0  
+matplotlib                3.2.2                         0  
+matplotlib-base           3.2.2            py38hef1b27d_0  
+mkl                       2020.1                      217  
+mkl-service               2.3.0            py38he904b0f_0  
+mkl_fft                   1.1.0            py38h23d657b_0  
+mkl_random                1.1.1            py38h0573a6f_0  
+ncurses                   6.2                  he6710b0_1  
+numpy                     1.19.0                   pypi_0    pypi
+olefile                   0.46                       py_0  
+openssl                   1.1.1g               h7b6447c_0  
+packaging                 20.4                       py_0  
+pandas                    1.0.5                    pypi_0    pypi
+parso                     0.7.0                      py_0  
+pcre                      8.44                 he6710b0_0  
+pexpect                   4.8.0                    py38_0  
+pickleshare               0.7.5                 py38_1000  
+pillow                    7.2.0            py38hb39fc2d_0  
+pip                       20.1.1                   py38_1  
+progressbar2              3.53.1                   pypi_0    pypi
+prompt-toolkit            3.0.5                      py_0  
+ptyprocess                0.6.0                    py38_0  
+pycparser                 2.20                     pypi_0    pypi
+pygments                  2.6.1                      py_0  
+pyparsing                 2.4.7                      py_0  
+pyqt                      5.9.2            py38h05f1152_4  
+pysynphot                 1.0.0                    pypi_0    pypi
+python                    3.8.3                hcff3b4d_2  
+python-dateutil           2.8.1                      py_0  
+python-utils              2.4.0                    pypi_0    pypi
+pytz                      2020.1                   pypi_0    pypi
+pyyaml                    5.3.1            py38h7b6447c_1  
+qt                        5.9.7                h5867ecd_1  
+readline                  8.0                  h7b6447c_0  
+requests                  2.24.0                   pypi_0    pypi
+scipy                     1.5.0            py38h0b6359f_0  
+secretstorage             3.1.2                    pypi_0    pypi
+sedkit                    1.0.6                    pypi_0    pypi
```

Comment:
for whatever reason pycharm can't find "import SEDkit", maybe case issue? We can remove anywho